iam-login-service/src/main/java/it/infn/mw/iam/authn/oidc/OidcExternalAuthenticationToken.java

```diff
@@ -52,14 +53,55 @@ public OidcExternalAuthenticationToken(OIDCAuthenticationToken authn, Date token
     super(authn, tokenExpiration, principal, credentials, authorities);
   }
 
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
Why we need to explicit this equals (+ hashCode)? I cannot find where it's used.